src/NServiceBus.Core/Routing/MessageDrivenSubscriptions/MessageDrivenSubscriptions.cs

```diff
@@ -8,6 +8,7 @@ namespace NServiceBus.Features
     using Unicast.Messages;
     using Unicast.Subscriptions.MessageDrivenSubscriptions;
 
+    //TODO: rename to MessageDrivenPublishSubscribe in next Major
```

Comment:
How about skipping the TODO and raise an issue? (Given that we're likely to merge this soon)